bridges/HashnodeBridge.php

```diff
@@ -0,0 +1,49 @@
+<?php
+class HashnodeBridge extends BridgeAbstract {
+
+	const MAINTAINER = 'liamka';
+	const NAME = 'Hashnode';
+	const URI = 'https://hashnode.com';
+	const URI_ITEM = 'https://hashnode.com';
+	const CACHE_TIMEOUT = 3600; // 1hr
+	const DESCRIPTION = 'See trending or latest posts in Hashnode community.';
+	const PARAMETERS = array();
+	const LATEST_POSTS = 'https://hashnode.com/api/stories/recent?page=';
+
+	public function collectData(){
+		$url = self::URI . '/' . $this->getInput('sort');
+
+		$html = getSimpleHTMLDOM($url)
+			or returnServerError('Error while downloading the website content');
+
+		$this->items = array();
+		$time = time();
+		for ($i = 0; $i < 5; $i++) {
+			$url = self::LATEST_POSTS . $i;
+			$content = file_get_contents($url);
```

Comment:
Use the helper function [`getContents()`](https://github.com/RSS-Bridge/rss-bridge/wiki/The-getContents-function) when fetching non-HTML data.
```suggestion
			$content = getContents($url)
				or returnServerError('Error fetching: ' . $url);
```